### DISCUSSION TITLE
How to add to an existing value using UPDATE?

### DISCUSSION THREAD

**Discussion by Guoyuer** [simolus3/drift#1140] [Q&A] [answered]
Is there any way to write SQL like this? update(table)..write() seems not proper
```
update products
set quantity = quantity +3```

**Comment by simolus3** [comment, marked as answer]
Sure:

```dart
update(table).write(YourTableCompanion.custom(quantity: table.quantity + const Constant<int>(3)));
```

**Reply by Guoyuer** [reply]
> Sure:
> 
> ```dart
> update(table).write(YourTableCompanion.custom(quantity: table.quantity + const Constant<int>(3)));
> ```

Great!
